libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeLegend/TreeLegend.tsx

```diff
@@ -87,24 +92,35 @@ export class TreeLegend extends React.Component<ITreeLegendProps> {
               <div className={classNames.metricBarRed} />
               <Stack tokens={cellTokens}>
                 <div className={classNames.smallHeader}>
-                  {localization.ErrorAnalysis.errorRate}
+                  {MetricUtils.getLocalizedMetric(
+                    this.props.selectedCohort.metricName,
+                    MetricLocalizationType.Name
+                  )}
                   <InfoCallout
-                    iconId={this._errorRateIconId}
-                    infoText={localization.ErrorAnalysis.errorRateInfo}
-                    title={localization.ErrorAnalysis.errorRateTitle}
+                    iconId={this._metricIconId}
```

Comment:
Is this id used anywhere?

Reply:
yes, it's the id in the IconButton, see InfoCallout.tsx:

        <IconButton
          id={this.props.iconId}